lib/linter.js

```diff
@@ -81,14 +79,14 @@ class Linter {
   _buildRule(ruleName, options, addToResults) {
     let loadedRules = this.config.loadedRules;
     let Rule = loadedRules[ruleName];
-
+    let configRule = this.config.rules[ruleName];
     let ruleOptions = Object.assign(
       {
         name: ruleName,
-        config: this.config.rules[ruleName],
+        config: configRule && configRule.config ? configRule.config : configRule,
```

Comment:
We should reset this change, the `processRules` method in `get-config.js` should **always** return the new structure (we should be "upgrading" any configs in the older format).

Reply:
yup... definitely, this was an oversight... meant to pass in just the standard format, but this got missed while i was working on another part. my bad :\